src/amp-story-player/amp-story-player-impl.js

```diff
@@ -68,6 +66,12 @@ const TOGGLE_THRESHOLD_PX = 50;
 /** @const {number} */
 const MAX_IFRAMES = 3;
 
+/** @enum {string} */
+const BUTTON_TYPES = {
+  BACK: 'back',
```

Comment:
We might want to prefix these, since they are going to be dispatched as events. We do not want to interfere with other events that the publisher might be using in their page. An example could be: `amp-story-player-back`.